### PR TITLE
Adds release checking to reload_deamon call

### PR DESCRIPTION
**Pull request by chuckbutler**

### CODE REVIEW

**Comment by chuckbutler** [comment]
related to #69

**Comment by chuckbutler** [comment]
Xenial:
PASS: 3 Total: 3 (431.003215 sec)
Trusty:
PASS: 3 Total: 3 (501.546384 sec)

File: reactive/docker.py [hunk 1]
```diff
@@ -152,5 +153,10 @@ def recycle_daemon():
...
     ''' Reload the system daemons from on-disk configuration changes '''
-    command = ['systemctl', 'daemon-reload']
-    check_call(command)
+    lsb = lsb_release()
+    code = lsb['DISTRIB_CODENAME']
+    if code != 'trusty':
+        command = ['systemctl', 'daemon-reload']
+        check_call(command)
+    else:
+        service_reload('docker')
```
Comment (mbruzek) [inline reactive/docker.py]: I have a strong feeling that running `service docker reload` would work on systemd computers as well, removing the need for this if statement. But as I don't have time to test this theory I will allow it. We should determine if reload works on xenial and come back to this in the future.

**Comment by mbruzek** [comment]
Fixes #74

**Comment by mbruzek** [comment]
+1

**Pull request merged**